scripts/addr2line.py

```diff
@@ -131,7 +133,7 @@ def __call__(self, address: str):
             return " ".join([self._binary, address, '\n'])
         # We print a dummy 0x0 address after the address we are interested in
         # which we can look for in _read_address
-        inputline = address + '\n0x0\n'
+        inputline = address + '\n\n'
```

Comment:
Sending an empty line to llvm-addr2line results in an ugly `llvm-addr2line: error: no module offset has been specified` being printed to stderr.

How about this?
```suggestion
        inputline = address + '\n,\n'
```

(By the way, that's what `perf` does. It uses a comma as the sentinel. See https://github.com/torvalds/linux/blob/eea6e4b4dfb8859446177c32961c96726d0117be/tools/perf/util/srcline.c#L729-L738.)

Reply:
> Sending an empty line to llvm-addr2line results in an ugly llvm-addr2line: error: no module offset has been specified being printed to stderr. (At least with llvm-18)

True. But since it's printed to `stderr` it's easily "filtarable".

> How about this?
> ...
This trick works and I can adjust the PR to use it (there is one more place where I'd need to add a coma: line 102). 
It's just this is another "black magic" undocumented thing that may break in some next release of llvm/binutils.

Reply:
Trick is applied